open-banking-accelerator/components/com.wso2.openbanking.accelerator.common/src/test/java/com/wso2/openbanking/accelerator/common/test/util/JWTUtilsTest.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
```

Comment:
Should be 2024

Reply:
Resolved in [7eaee49](https://github.com/ImalshaD/financial-open-banking/pull/6/commits/7eaee4933908991557b77c90ffa5cb31dc2854c5)